src/main/java/org/aludratest/eclipse/vde/internal/editors/VisualDataEditorPage.java

```diff
@@ -390,6 +406,32 @@ public String getText(Object element) {
 		}
 	}
 
+	static class ExternalTestIDValidator implements IInputValidator {
+
+		private List<String> disallowedIDs;
+
+		public ExternalTestIDValidator(List<String> disallowedIDs) {
+			// no copy constructor as this is all internal and under our control
+			this.disallowedIDs = disallowedIDs;
+		}
+
+		@Override
+		public String isValid(String newText) {
+			if (newText == null || newText.length() == 0) {
+				return "Please enter a value.";
+			}
+			if (newText.trim().length() != newText.length()) {
+				return "Please do not begin or end with a space.";
+			}
+
+			if (disallowedIDs.contains(newText)) {
+				return "There already exists a configuration with this ID";

```

Comment:
Correct would be, "with this external test ID" - but, as stated earlier, just remove the whole validator.